src/components/NewCommentForm.tsx

```diff
@@ -14,24 +89,32 @@ export const NewCommentForm: React.FC = () => {
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={classNames('input', {
+              'is-danger': !!nameError,
```

Comment:
why do you use !! when you already have a boolean value? Here and in some other places